public/js/script.js

```diff
@@ -99,6 +105,29 @@ function mountPlots(screenData, sampleData, featureNames) {
     var neighbourPlot = new NeighbourPlot(sampleData, '#neighbourplot');
     var filter = new Filter(sampleData, neighbourPlot);
 
+    // attach behaviour to backwards and forwards buttons, unhide them
+    $backButton.unbind('click');
+    $forwardButton.unbind('click');
+
+    $backButton.on('click', function() {
+        var backId = history.back();
+        if(backId) {
+            lineplot.getSampleData(backId);
+            neighbourPlot.updatePoint(backId);
+            neighbourImages.getImages(backId);
```

Comment:
Hmm, another set of things that are all updated together. Perhaps worth having a more global function that updates the currently-selected sample globally? (ie in all models?) Soon the histogram will also need to be updated...